### PR TITLE
My portfolio

### PR DESCRIPTION
**Pull request by Abel-desu**
adding some features on it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+            A daily selection of privately<br />personalized reads; 
+            no accounts or<br /> sign-ups required.
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action">
+            <p>See Project</p>
+          </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button ` See project` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
figma

File: index.html [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                    no accounts or<br /> sign-ups required.
+                  </p>
+                  <ul>
+                    <li><a href="#">html</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                  </ul>
+          <div class="action1">
+            <p>See Project</p>
+          </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button ` See project` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
figma

File: index.html [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+            no accounts or<br /> sign-ups required.
+          </p>
+          <ul>
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+          </ul>
+          <div class="action2">
+            <p>See Project</p>
+          </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button ` See project` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
figma

File: index.html [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+          no accounts or<br /> sign-ups required.
+      </p>
+     <ul>
+        <li><a href="#">html</a></li>
+        <li><a href="#">css</a></li>
+        <li><a href="#">javascript</a></li>
+      </ul>
+      <div class="action3">
+        <p>See Project</p>
+      </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button ` See project` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
figma

File: index.html [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+        <div class="primary5">
+          <p>
+            Hello I am a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, do not hesitate to contact me.
+          </p>
+        </div>
+        <div class="connect"> 
+          <p>LET US CONNECT</p>
+        </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Kindly wrap this `p` tag with `hx` tag for HTML  good practice.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+                <img class="a1-icon" src="./images/angellist.png" alt="a-icon">
+              </a>
+            </li>
+           </ul>
+          
+        </div>
+        <div class="action4">
+          <p>Get my Resume</p>
+        </div>
+      </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] The button ` Get my Resume` should be a link or button tag, and follow the button styling: Active, Hover, and Pressed (see design). Kindly implement it
### Figma button design
figma

File: index.html [hunk 7]
```diff
@@ -0,0 +1,290 @@
...
+      <div class="skill-list">
+        <div class="frame12">
+          <div class="language">
+            <span class="text">Languages</span>
+            <span class="down-angle"><i class="fa-solid fa-angle-down"></i></span>
```
Comment (roshan-bajgain) [inline index.html:264]: - [ ] Kindly wrap this `languages` with ul and put this `language` in li tag for better practice

File: index.html (hunk 1 above)
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you to stick as much as possible with Figma design.
- [ ] Kindly implement in all the section.
### your design
his
### Figma design
figma-design

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @Abel-desu, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**